test/integration/auth.spec.ts

```diff
@@ -1643,6 +1707,7 @@ describe('admin.auth', () => {
         multiFactorConfig: deepCopy(expectedUpdatedTenant2.multiFactorConfig),
         // Test clearing of phone numbers.
         testPhoneNumbers: null,
+        recaptchaConfig: deepCopy(expectedUpdatedTenant2.recaptchaConfig),
```

Comment:
Does updating a config with an `undefined` `recaptchaConfig` deletes the current config? Do we need to add a test (unit or integration) for this scenario?

Reply:
Good point.  With recaptchaConfig being undefined, the current config will stay unchanged.  Added integration test for updateTenants.  As of project config, since it's the only config, it got covered by unit test.